Added LoopTimeComponent and VoltageCompensatingMotor to docs

AchintyaAkula opened the pull request:
Added voltage compensating motor, example is slightly lengthy, not sure how it will display.
Also added Looptime Component to components list

AchintyaAkula committed:
Added docs for VoltageCompensatingMotor

Added docs for voltage compensating motor, although example is little long, not sure how it will display

AchintyaAkula committed:
Added LoopTimeComponent to components list

AchintyaAkula committed:
Updated FeedbackCRServo usage example to match current constructors

AchintyaAkula committed:
Added LoopTimeComponent to TeleOp guide

Added LoopTimeComponent part to TeleOp section of guide

zachwaffle4 commented on src/nextftc/hardware/motors-and-servos/motors.md:
```diff
@@ -102,4 +102,40 @@ MotorGroup myMotorGroup = new MotorGroup(
+
+// or with "ideal" voltage(Volts) and voltage caching time(seconds) specifications
+VoltageCompensatingMotor myVoltageCompensatingMotor = new VoltageCompensatingMotor(myMotorEx, 0.5, 12.0)
+VoltageCompensatingMotor myVoltageCompensatingMotor = new VoltageCompensatingMotor(myMotorEx, "0.5", 12.0)
```
`"0.5"` is not a valid duration string. `"500ms"` and `"0.5s"` would be correct here.

AchintyaAkula replied:
Got it. Just fixed it to have `500ms` instead, and removed the kotlin example as mentioned above that uses a string. Please review my changes

AchintyaAkula committed:
Fixed duration formats in examples

Updated examples of VoltageCompensatingMotor initialization to use numeric values instead of strings for voltage caching time.
